packages/run-protocol/test/vaultFactory/test-vault-interest.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
I've been thinking the other direction, to disable the rule in tests. The rule was created because,
> performing an await as part of each operation is an indication that the program is not taking full advantage of the parallelization benefits of async/await

but that's less pertinent to test code. In test code clarity of what's happening is more important than small perf wins. Also we often use loops like this to do data-driven coding that isn't necessarily parallelizable. E.g. in the await on 115.

So I don't think there's anything to _fix_ about allowing an await in that loop. If pressed we could refactor like so,
```
  async function checkInterest(i, charge) {
    await testJig.advanceRecordingPeriod();
    interest += charge;
    t.is(
      vault.getCurrentDebt().value,
      startingDebt + interest,
      `interest charge ${i} should have been ${charge}`,
    );
    t.is(vault.getNormalizedDebt().value, startingDebt);
  };
  await checkInterest(1, 3n);
  await checkInterest(2, 4n);
  await checkInterest(3, 4n);
  await checkInterest(4, 4n);
```
but I think that's worse.

Reply:
I'm not the one that turned on `no-await-in-loop`, so I don't feel like I can approve that choice. An issue to discuss whether the rule is superfluous in tests would satisfy me.

Reply:
Fair!